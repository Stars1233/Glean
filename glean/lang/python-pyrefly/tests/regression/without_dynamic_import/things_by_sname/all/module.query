query: python.ModuleBySName K -> V; {K, V}
transform: [normord, []]
