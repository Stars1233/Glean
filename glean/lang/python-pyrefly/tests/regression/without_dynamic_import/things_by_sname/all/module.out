[
  "@generated",
  {
    "key": {
      "tuplefield0": { "key": { "local_name": { "key": "a" } } },
      "tuplefield1": { "key": { "name": { "key": "a" } } }
    }
  },
  {
    "key": {
      "tuplefield0": {
        "key": {
          "local_name": { "key": "b" },
          "parent": { "key": { "local_name": { "key": "mod" } } }
        }
      },
      "tuplefield1": { "key": { "name": { "key": "mod.b" } } }
    }
  },
  {
    "key": {
      "tuplefield0": { "key": { "local_name": { "key": "mod" } } },
      "tuplefield1": { "key": { "name": { "key": "mod" } } }
    }
  }
]